src/map/atcommand.c

```diff
@@ -7746,10 +7748,11 @@ ACMD(leave) {
 }
 
 ACMD(accept) {
-	if (!duel->checktime(sd)) {
+	int64 diff = duel->checktime(sd);
+	if (diff > 0) {
 		char output[CHAT_SIZE_MAX];
-		// "Duel: You can take part in duel only one time per %d minutes."
-		sprintf(output, msg_fd(fd,356), battle_config.duel_time_interval);
+		// "Duel: You can take part in duel again after %d seconds."
+		sprintf(output, msg_fd(fd,356), diff);
```

Comment:
`duel->checktime()` current return int64 ... the `txt_time` use `int` ... 
beside I believe that nobody ever set the duel settings to higher than 1 minute, I am sure they set it as less as possible, so the minute or hour or even day display are kinda useless, and hence unnecessary processing.